spec/support/requests_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module RequestsHelper
+  # @param email [String]
+  # @param password [String]
+  # @param request_token [String]
+  def send_sign_in_request(email, password, request_token)
```

Comment:
We should ditch this for `Devise::IntegrationHelpers`. You only want to make this request when you test the sign in flow, otherwise we should just use the Devise helper.